pkgs/top-level/all-packages.nix

```diff
@@ -18718,6 +18718,8 @@ in
 
   chromium = callPackage ../applications/networking/browsers/chromium (config.chromium or {});
 
+  chromiumVaapi = chromium.override { useVaapi = true; };
```

Comment:
Maybe?
```suggestion
  chromiumWithVaapi = chromium.override { useVaapi = true; };
```

Reply:
If I name this as `chromiumWihVaapi`, package will be named like `chromiumWithVaapi` and in my opinion is wrong. In other distributives it named like `chromium-vaapi`.

Reply:
Probably I should rename it to `chromium-vaapi`?